guides/common/modules/proc_installing-webhooks-shellhooks-plugin.adoc

```diff
@@ -0,0 +1,28 @@
+[id="installing-webhooks-shellhooks-plugin_{context}"]
+= Installing the Webhooks Plugin
+
+Use the following procedure to install the webhooks plugin. 
+After installing the `foreman_webhooks` plugin, you can configure {ProjectServer} to send webhook requests.
+
+.Procedure
+
+* Install the webhooks plugin using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --enable-foreman-plugin-webhooks
+----
+
+* Optionally, install the shellhooks plugin on each {SmartProxy} used for shellhooks, using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --enable-foreman-proxy-plugin-shellhooks
+----
+
+* Optionally, install the CLI plugin using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# yum install tfm-rubygem-hammer_cli_foreman_webhooks
```

Comment:
Currently it's the only way, but there is a set of patches to allow users to install the hammer plugin via installer as well.